src/consts.ts

```diff
@@ -65,14 +65,14 @@ export const SIDEBAR: Sidebar = {
     Usage: [{ text: "Gaming on Ultramarine", link: "en/usage/gaming" }],
 
     "Release Notes": [{ text: "Errata", link: "en/release/errata" }],
-    // Chromebook: [
-    //   { text: "Getting Started", link: "en/chromebook/chromebook" },
-    //   { text: "Getting Chromebook Edition", link: "en/chromebook/getting" },
-    //   { text: "Installing to a Chromebook", link: "en/chromebook/preparing" },
-    //   { text: "Post-Install", link: "en/chromebook/postinstall" },
-    //   { text: "ARM Chromebooks (Beta)", link: "en/chromebook/armbeta" },
-    //   { text: "Terminology", link: "en/chromebook/terminology" },
-    // ],
+    Chromebook: [
+      { text: "Getting Started", link: "en/chromebook/chromebook" },
+      { text: "Getting Chromebook Edition", link: "en/chromebook/getting" },
```

Comment:
This is bad because there should be more meows.
```suggestion
      { text: "Getting meowbook Edition", link: "en/chromebook/getting" },
```